src/components/Taxonomy/Taxonomy.tsx

```diff
@@ -112,21 +135,27 @@ function isSubArray(item: string[], parent: string[]) {
   return parent.every((n, i) => item[i] === n);
 }
 
+
 // @todo change `flat` into `alwaysOpen` and move it to context
-const Item = ({ item, isFiltering }: { item: TaxonomyItem, isFiltering?: boolean }) => {
+const Item: React.FC<any> = (props: RowProps) => {
```

Comment:
first parameter of `React.FC` generic is for props, so it should be like `Item: React.FC<RowProps> = (props) => ...`